modules/store/spec/edge.spec.ts

```diff
@@ -21,11 +21,17 @@ describe('ngRx Store', () => {
     let store: Store<TodoAppSchema>;
 
     beforeEach(() => {
-      const injector = createInjector(
-        StoreModule.forRoot<TodoAppSchema>({ todos, todoCount } as any)
-      );
+      try {
+        TestBed.resetTestingModule();
```

Comment:
When running these tests via `jasmine_node_test` the `TestBed` is not automatically reset.

Reply:
I don't know much about TestBed, but I would expect `jasmine_node_test` just evaluates all scripts, and wouldn't do anything special to reset test state - that would be the job of whatever framework you are testing?

Reply:
Can you help me understand what layer was doing this job before?

Reply:
Currently Angular's core testing library owns resetting the testing module: https://github.com/angular/angular/blob/master/packages/core/testing/src/before_each.ts#L25

I'm assuming that the `beforeEach` hook it is calling belongs to a different Jasmine environment.

Reply:
I was able to confirm that the Jasmine environment I'm working with in the bootstrap script is different from the one being used when running my tests. I wasn't able to add a custom reporter in bootstrap, for instance.